fedimint-server/src/config/api.rs

```diff
@@ -481,6 +486,65 @@ impl ConfigGenApi {
             sleep(Duration::from_millis(100)).await;
         }
     }
+
+    // Check the status of the bitcoin rpc connection
+    pub async fn check_bitcoin_status(&self) -> ApiResult<BitcoinRpcConnectionStatus> {
+        let bitcoin_rpc_config = BitcoinRpcConfig::get_defaults_from_env_vars().map_err(|e| {
```

Comment:
This config might not match what's actually running since `FM_FORCE_*` takes precedence.

https://github.com/fedimint/fedimint/blob/91c30355491511fe634703d085566c52d6bc0c28/fedimint-bitcoind/src/lib.rs#L70

Reply:
@Kodylow

Reply:
Call: related to #5993

Reply:
This was super easy to address actually so added in next commit